packages/ui-components/src/Dropzone.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { IDragEvent } from '@lumino/dragdrop';
+import React, { useCallback, useEffect, useRef } from 'react';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/interface-name-prefix
```

Comment:
We should not be disabling lint unless absolutely necessary, based on our lint rules the following interface should be called `IHTMLElementEventMap`, is there a reason it can't be?

Reply:
@ajbozarth We are overriding the global `HTMLElementEventMap` not creating a new interface

Reply:
The `HTMLElementEventMap` type doesn't have `lm-dragenter`, `lm-drop`, etc.. so this just adds those to the type so we don't get typescript errors

Reply:
gotcha, ok the this still LGTM